cognite/client/_api/assets.py

```diff
@@ -147,10 +147,10 @@ def list(
         parent_ids: List[int] = None,
         root_ids: List[int] = None,
         root_external_ids: List[str] = None,
-        metadata: Dict[str, Any] = None,
+        metadata: Dict[str, str] = None,
```

Comment:
we do allow other types in metadata values even if they're persisted as strings, no?

Reply:
While we do allow it I guess this shouldn't be encouraged? I don't know of anywhere we use any sort of implisit casting our self, and then I guess it is good manners to suggest users serializing before putting into the dictionary, rather than rely on us doing it the way they would expect? Type hints are not enforced, so if someone is adventurous this shouldn't break anything at least.

Reply:
we do allow this in e.g. all filters and it's pretty standard, since people don't want to import 25 different objects for every simple call